packages/arb-validator/rollup/listener.go

```diff
@@ -43,9 +48,8 @@ type ChainListener interface {
 	MessageDelivered(context.Context, *ChainObserver, arbbridge.MessageDeliveredEvent)
 
 	AssertionPrepared(context.Context, *ChainObserver, *preparedAssertion)
-	ValidNodeConfirmable(context.Context, *ChainObserver, *confirmValidOpportunity)
-	InvalidNodeConfirmable(context.Context, *ChainObserver, *confirmInvalidOpportunity)
-	PrunableLeafs(context.Context, *ChainObserver, []pruneParams)
+	NodesConfirmable(context.Context, *ChainObserver, *valprotocol.ConfirmOpportunity)
+	PrunableLeafs(context.Context, *ChainObserver, []valprotocol.PruneParams)
```

Comment:
The naming here is inconsistent.  NodesConfirmable is noun-adjective.  PrunableLeafs is adjective-noun.  Should be consistent.

Reply:
Fixed